test/analysis/alias/array-arguments-const-ref.chpl

```diff
@@ -0,0 +1,17 @@
+config const n = 10;
+
+proc main() {
+  var A: [1..10] int = 1;
+  var B: [1..10] int = 2;
+
+  var sum = test(A, B);
+  writeln(sum);
+}
+
+proc test(const ref a, const ref b) {
```

Comment:
If there isn't a test for this already, could you add one that has multiple levels of function calls where the outer-most call creates an alias? Something like:

```chpl
test2(A, B); // and a test with test2(A, A)

proc test2(ref a, ref b) {
  test1(a, b);
}

proc test1(ref a, ref b) {
  test(a, b);
}
```

Reply:
I added such tests (array-arguments-ref-nested-*.chpl).